src/DynamoCoreWpf/UI/GuidedTour/Step.cs

```diff
@@ -625,6 +642,28 @@ internal void HighlightMenuItem(MenuItem highlighMenuItem, bool bVisible)
             }
         }
 
+        /// <summary>
+        /// This method will call the collapseExpandPackage javascript method with reflection, so the package expander in LibraryView will be clicked
+        /// </summary>
+        internal void CollapseExpandPackage()
```

Comment:
Same here

Reply:
@filipeotero this method cannot be removed or merged with other method since is called in a Popup  when the Button.Click event is executed and at this moment in the framework we don't have the functionality for add handlers that execute js methods to events (I think ManageEventHandler method is not working for Buttons inside the Popup). Thanks